ska-sa/katgui
# Hotfix V27: Download proper zero values on sensor-graph csv download

#### Download proper values for zero
Make the "download_csv" functionality of sensor-graph append proper values for `0` when there is no "Resampling_Interval"

The problem was: If `No Sampling Interval` is set, values equal to `0` were downloaded as `undefined`.
We fixed the logic

#### Testing for System-Team
Under Sensor-Graph on KatGui
- Search for example `m000_dig_l_band_noise_diode`, set 1 minute Interval(or any) and plot
- Check the buttons as listed below and click "Download As CSV"
    - `Use Unix timestamps` + `Include Value timeStamps` + `1 minute average`
    - `Include Value timeStamps` + `1 minute average`
    - `Use Unix timestamps` + `1 minute average`
    - `1 minute average`
- With no Interval
    - `Use Unix timestamps` + `Include Value timeStamps` + `No Interval`
    - `Include Value timeStamps` + `No Interval`
    - `Use Unix timestamps` + `No Interval`
    - `No Interval`

See [MT-1675](https://skaafrica.atlassian.net/browse/MT-1675)
